src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.props

```diff
@@ -19,4 +20,19 @@
   <PropertyGroup>
     <GetNuGetPackageVersionsDependsOn>CreateVersionFileDuringBuild</GetNuGetPackageVersionsDependsOn>
   </PropertyGroup>
+
+  <!-- Common BuildTools properties -->
+  <PropertyGroup>
```

Comment:
Thanks for starting the migration of a number of these common properties. I think we should clean out a lot more of the other properties as well but that can be done as a separate work item.

cc @crummel @markwilkie